lms/djangoapps/commerce/models.py

```diff
@@ -23,6 +26,18 @@ class Meta(object):
         default='/basket/single-item/',
         help_text=_('Path to single course checkout page hosted by the E-Commerce service.')
     )
+    cache_ttl = models.PositiveIntegerField(
```

Comment:
@ahsan-ul-haq you may want to open a PR against https://github.com/edx/app-permissions so that we're able to get permissions to set this field in stage/prod.